core/src/main/java/contacts/core/Contacts.kt

```diff
@@ -137,6 +140,11 @@ interface Contacts {
      * Registry for all [CrudApi.Listener]s.
      */
     val apiListenerRegistry: CrudApiListenerRegistry
+
+    /**
+     * Registry for [Logger]
+     */
+    var loggerRegistry: LoggerRegistry
```

Comment:
This should be a `val`. I'll make the change after merging this PR.

Reply:
I made the update here https://github.com/vestrel00/contacts-android/commit/e321874e5c37739a6cb0a26ac479400a662b911e